added warning and timestamp for messages to delta function

Sailoshi opened the pull request:
made some changes on delta function

Sailoshi committed:
added warning and timestamp for messages to delta function

Commit:
changed log message

Chris-Graffagnino reviewed (approved):
Thank You! Approved!

Chris-Graffagnino committed:
Merge branch 'master' into master

Phillip-Sailer committed:
replaced exit with return to avoid closing shell

Phillip-Sailer committed:
added error message when node is not running and current blockheight couldn't be evaluated.

Phillip-Sailer committed:
changed text

Commit:
added new function time to next slot production (next)

Commit:
added missing leaders function

Commit:
refactored leaders function (duplicated code)

Chris-Graffagnino committed:
Update .bash_profile

Added pheelLikeWater's updates to delta function, along with his new function, next. Added references to leader_logs function.
